### PR TITLE
Introduce markdown Blockquote component. #613

### PR DESCRIPTION
**Pull request by ryanoglesby08**
So that it is easier for consumers to customize their styleguide by replacing
components.

### CODE REVIEW

File: src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Styled from 'rsg-components/Styled';
+import { styles as paraStyles } from 'rsg-components/Para';
+
+const styles = ({ space, color, fontSize, fontFamily }) => ({
+	blockquote: {
+		...paraStyles({ space, color, fontFamily }).para,
```
Comment (sapegin) [inline src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js]: I think this is confusing: you're overwriting half of the styles, I’d just copy the relevant styles here.

File: src/rsg-components/Markdown/Blockquote/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './BlockquoteRenderer';
```
Comment (sapegin) [inline src/rsg-components/Markdown/Blockquote/index.js]: Should be `rsg-components/Markdown/Blockquote`.

File: src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js (hunk 1 above)
Comment (ryanoglesby08) [reply src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js]: Sure, can do. I was just copying the existing styles from the old style-based way in Markdown.js. Can switch to not using Para at all here.

File: src/rsg-components/Markdown/Blockquote/index.js (hunk 2 above)
Comment (sapegin) [reply src/rsg-components/Markdown/Blockquote/index.js]: If you see this pattern somewhere in Styleguidist, then it's a bug ;-) That prevents component overriding to work properly, so we should always use the full form.

File: src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js (hunk 1 above)
Comment (sapegin) [reply src/rsg-components/Markdown/Blockquote/BlockquoteRenderer.js]: It was a bit better when it was defined in the same file ;-)

File: src/rsg-components/Markdown/Blockquote/index.js (hunk 2 above)
Comment (ryanoglesby08) [reply src/rsg-components/Markdown/Blockquote/index.js]: Ahhhhhh I see now. I knew I wasn't crazy! :) I missed this commit that changed a lot of the exports from a relative path to the rsg-components version: 

https://github.com/styleguidist/react-styleguidist/commit/46156bb932121df5a176852d0fcdfd43cad83ef3

Following suit now

**Comment by sapegin** [comment]
Thanks, merged!

**Pull request merged**